addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2863,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#670"
-msgid "Verbose logging for CURL library (http, dav)"
+msgid "Verbose logging for \"CURL\" library \"HTTP(S), DAV\""
 msgstr ""
 
 #empty string with id 671
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#672"
-msgid "Verbose logging for FFMPEG libraries"
+msgid "Verbose logging for \"FFmpeg\" libraries"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#673"
-msgid "Verbose logging for RTMP library"
+msgid "Verbose logging for \"RTMP\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#674"
-msgid "Verbose logging for DBUS calls"
+msgid "Verbose logging for \"DBUS\" calls"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#675"
-msgid "Verbose logging for JSON-RPC requests"
+msgid "Verbose logging for \"JSON-RPC\" requests"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#676"
-msgid "Verbose logging for AUDIO component"
+msgid "Verbose logging for \"AUDIO\" component"
```

Comment:
Yes, I think so also. the verbose logging has all been capitalized originally,.. The reason for quotes something needs to separate them from sentence, otherwise it's \"Audio\" or audio in here.